src/components/MyKiva/BadgeTile.vue

```diff
@@ -131,6 +131,12 @@ const selectedTier = computed(() => {
 });
 
 const badgeName = computed(() => selectedTier?.value?.badge?.challengeName ?? '');
+
+const tileTitle = computed(() => {
+	const tierLevel = selectedTier?.value?.tier?.level ?? '';
```

Comment:
I think we want to use `levelName` instead of the level number.

Reply:
So, we'll show smt like `(level ✨100✨)`?
Not sure if precisely a level...

Reply:
That's how we show the level other locations, like the journey modal. You can ask product in our team channel.